index.js

```diff
@@ -330,7 +340,32 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.keys(pattern, handleResponse(conn, cb));
+
+      // Use an object to dedupe as scan can return duplicates
+      var keysObj = {};
+      var count = 100;
```

Comment:
Sorry, I said `del()`, but I meant `keys()`. Do you guys have any input on how to make the scan count an option? Also, the default value for redis is 10. I chose 100 as a balance between small and large results.